src/ChirpWeb/Program.cs

```diff
@@ -64,17 +78,28 @@
 
 app.MapRazorPages();
 
+// From ChatGPT to ensure the database is up to date with migrations when run
```

Comment:
This is outcommented? Is it something we need later?

Reply:
I don't think so, I'll remove it completely